src/auth/auth.ts

```diff
@@ -211,80 +211,96 @@ export const loadAuthToken = async (
     authTokenState: loadedAuthTokenState,
   }));
   if (loadedAuthTokenState.status === 'success') {
-    const oidcUser: OidcUser = loadedAuthTokenState.oidcUser;
```

Comment:
All I did was move each of these `if/else` cases to their own functions. This will make the code more testable